tests/kafka_consumer_manager/test_copy_group.py

```diff
@@ -47,7 +47,26 @@ def mock_kafka_info(self, topics_partitions):
             mock_ZK.return_value.__enter__.return_value = mock_ZK
             yield mock_process_args, mock_user_confirm, mock_ZK
 
-    def test_run(self, mock_client):
+    def test_run_with_kafka(self, mock_client):
+        topics_partitions = {
+            "topic1": [0, 1, 2],
+            "topic2": [0, 1]
+        }
+        with self.mock_kafka_info(
+            topics_partitions
+        ) as (mock_process_args, mock_user_confirm, mock_ZK), \
+            mock.patch('kafka_utils.kafka_consumer_manager.commands.'
+                       'copy_group.get_current_consumer_offsets') \
+            as mock_get_current_consumer_offsets, \
+            mock.patch('kafka_utils.kafka_consumer_manager.commands.'
+                       'copy_group.set_consumer_offsets') as mock_set_consumer_offsets:
+            cluster_config = mock.Mock(zookeeper='some_ip')
+            args = mock.Mock(source_groupid='old_group', dest_groupid='new_group', storage='kafka')
+            CopyGroup.run(args, cluster_config)
+            assert mock_set_consumer_offsets.call_count == 1
+            assert mock_get_current_consumer_offsets.call_count == 1
```

Comment:
This is a unit test, so I am relying on the underneath functions in offsets.py to be properly working. I have added param check to verify that set/get consumer offsets is being called with correct values. There are acceptance tests which should be covering the broader cases.